samples/namespacescope-sample/createUIDefinition.json

```diff
@@ -0,0 +1,247 @@
+								}
+							},
+							{
+								"name": "newClusterResourceName",
+								"type": "Microsoft.Common.TextBox",
+								"label": "AKS cluster name",
+								"defaultValue": "",
+								"toolTip": "Use only allowed characters",
+								"constraints": {
+									"required": true,
+									"regex": "^[a-z0-9A-Z]{6,30}$",
+									"validationMessage": "Only alphanumeric characters are allowed, and the value must be 6-30 characters long."
+								}
+							},
+							{
+								"name": "kubernetesVersion",
+								"type": "Microsoft.Common.DropDown",
+								"label": "Kubernetes version",
+								"toolTip": "The version of Kubernetes that should be used for this cluster. You will be able to upgrade this version after creating the cluster.",
+								"constraints": {
+									"allowedValues": "[map(steps('clusterDetails').newClusterSection.aksVersionLookupControl.properties.orchestrators, (item) => parse(concat('{\"label\":\"', item.orchestratorVersion, '\",\"value\":\"', item.orchestratorVersion, '\"}')))]",
+									"required": true
+								}
+							},
+							{
+								"name": "vmSize",
+								"type": "Microsoft.Compute.SizeSelector",
+								"label": "VM size",
+								"toolTip": "The size of virtual machine for VM.",
+								"recommendedSizes": [
+									"Standard_B4ms",
+									"Standard_DS2_v2",
+									"Standard_D4s_v3"
+								],
+								"constraints": {
+									"allowedSizes": [
+										"Standard_B4ms",
+										"Standard_DS2_v2",
+										"Standard_D4s_v3"
+									],
+									"excludedSizes": []
+								},
+								"osPlatform": "Linux"
+							},
+							{
+								"name": "enableAutoScaling",
+								"type": "Microsoft.Common.CheckBox",
+								"label": "Enable auto scaling",
+								"toolTip": "Enable auto scaling",
+								"defaultValue": true						
+							},
+							{
+								"name": "vmCount",
+								"type": "Microsoft.Common.Slider",
+								"min": 1,
+								"max": 10,
+								"label": "VMCount",
+								"subLabel": "",
+								"defaultValue": 1,
+								"showStepMarkers": false,
+								"toolTip": "Specify VM count",
+								"constraints": {
+									"required": false
+								},
+								"visible": true
+							}
+						],
+						"visible": "[basics('createNewCluster')]"
+					}
+				]
+			},
+			{
+				"name": "extensionDetails",
+				"label": "Cluster Extension Details",
+				"elements": [
+					{
+						"name": "extensionResourceName",
+						"type": "Microsoft.Common.TextBox",
+						"label": "Cluster extension resource name",
+						"defaultValue": "",
+						"toolTip": "Only lowercase alphanumeric characters are allowed, and the value must be 6-30 characters long.",
+						"constraints": {
+							"required": true,
+							"regex": "^[a-z0-9]{6,30}$",
+							"validationMessage": "Only lowercase alphanumeric characters are allowed, and the value must be 6-30 characters long."
+						},
+						"visible": true
+					},
+					{
+						"name": "targetNamespace",
```

Comment:
>targetNamespace [](http://example.com/codeflow?start=15&length=15)

I think namespace can be provided for cluster scoped extensions as well. Please check below

[https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions#extension-scope](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions#extension-scope)

Also, in our packaging manifest, namespace details are optional. Check below.

[https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux#create-the-manifest-file](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux#create-the-manifest-file)